main.py

```diff
@@ -552,6 +552,11 @@ def delete_metadata(self, request: Request) -> JSONResponse:
     def redeploy(self, request: Request) -> JSONResponse:
         """Endpoint to force the redeployment of an EVC."""
         circuit_id = request.path_params["circuit_id"]
+        try_avoid_same_s_vlan = request.query_params.get("avoid_vlan", "true")
```

Comment:
query_params is still using `avoid_vlan` instead of `try_avoid_same_s_vlan` that you answered in other comments. Let's also enhance the unit test cov for this endpoint with this new arg.

Reply:
Changed from `avoid_vlan` to `try_avoid_same_s_vlan`